Makefile

```diff
@@ -221,7 +223,7 @@ format:
 ###                                Localnet                                 ###
 ###############################################################################
 
-start-localnet-ci:
+start-localnet-ci: build
 	./build/gaiad init liveness --chain-id liveness --home ~/.gaiad-liveness
```

Comment:
Would like to add a `rm -rf here ~/.gaiad-liveness`, opinions?

Reply:
maybe not ? in case people still want to keep the data.  let them manually `rm`

Reply:
liveness test shouldn't ever be persisted though. it's just to make sure that blocks are produced. i'm ok with removing as long as we're sure there is no chance liveness tests are being configured with custom home directories or anything like that, right?

Reply:
We do use a custom home, but this target is really just for the CI & occasional local testing

Reply:
ok let's add the `rm`